LumpNav/LumpNav.py

```diff
@@ -261,11 +267,16 @@ def setupScene(self): #applet specific
       self.needleModelToNeedleTip.SetName("NeedleModelToNeedleTip")
       m = vtk.vtkMatrix4x4()
       m.SetElement( 0, 0, 0 )
-      m.SetElement( 1, 1, 0 )

```

Comment:
It seems that you need different values, so this matrix should be configurable, similarly to the IP address of the PlusServer (stored in the Slicer.ini file). There are already helper functions to read/write matrices from the ini file (used for storing pivot calibration results).